.github/workflows/pr.yaml

```diff
@@ -0,0 +1,80 @@
+name: pr
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-builder:
+    needs:
+      - checks
+      - conda-python-build
+      - conda-python-tests
+      - wheel-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@main
+  checks:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@main
+  conda-python-build:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/conda-python-matrix-build.yaml@main
+    with:
+      build_type: pull-request
+  conda-python-tests:
+    needs: conda-python-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/conda-python-tests.yaml@main
+    with:
+      build_type: pull-request
+  wheel-build:
+    strategy:
+      fail-fast: false
+    runs-on:
+      - self-hosted
+      - linux
+      - amd64
+      - cpu4
+    container:
+      image: rapidsai/ci:latest
+      env:
+        RAPIDS_BUILD_TYPE: 'pull-request'
+        NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Standardize repository information
+        run: |
+          echo "RAPIDS_REPOSITORY=${{ github.repository }}" >> "${GITHUB_ENV}"
+          echo "RAPIDS_SHA=$(git rev-parse HEAD)" >> "${GITHUB_ENV}"
+          echo "RAPIDS_REF_NAME=${{ github.ref_name }}" >> "${GITHUB_ENV}"
+          echo "RAPIDS_BUILD_TYPE=pull-request" >> "${GITHUB_ENV}"
```

Comment:
I'm probably going to create a composite action with inputs to consolidate this logic into a single location. I don't like that it's copy/pasted in this repo and others at the moment.

Reply:
I won't let this hold up the PR. We can update it in a later PR.

Reply:
turns out these vars aren't needed. removed in 8a07c21 (details in the commit body)